resource/traversers/dfu_impl.cpp

```diff
@@ -606,7 +606,15 @@ int dfu_impl_t::dom_dfv (const jobmeta_t &meta, vtx_t u,
     for (auto &resource: resources) {
         if ((resource.type == (*m_graph)[u].type) &&
             (!resource.label.empty())) {
-            (*m_graph)[u].idata.ephemeral.insert (m_best_k_cnt, "label", resource.label);
+            if ((*m_graph)[u].idata.ephemeral.insert (m_best_k_cnt,
+                                                      "label",
+                                                      resource.label) < 0) {
+                m_err_msg += "dom_dfv: inserting label into ephemeral failed.\n";
+                m_err_msg += strerror (errno);
+                m_err_msg += ".\n";
+                rc = -1;
```

Comment:
The pattern throughout `dfu_impl.cpp` is to set the return code in the condition.  Unfortunately, my suggestion extends beyond 80 characters on one line, and I'm not sure the best way to split such a short line. 
```suggestion
            if ( (rc = (*m_graph)[u].idata.ephemeral.insert (m_best_k_cnt,
                                                             "label",
                                                             resource.label)) < 0) {
                m_err_msg += "dom_dfv: inserting label into ephemeral failed.\n";
                m_err_msg += strerror (errno);
                m_err_msg += ".\n";
```

Reply:
I ended up just splitting is across two lines (and dropping the rc = -1):

```
           rc = (*m_graph)[u].idata.ephemeral.insert (m_best_k_cnt,
                                                       "label",
                                                       resource.label);
            if (rc < 0) {
```